channels/factories.py

```diff
@@ -17,13 +17,37 @@
 
 FAKE = faker.Factory.create()
 
-Channel = namedtuple('Channel', ['name', 'title', 'channel_type', 'public_description', 'api'])
 
-TextPost = namedtuple('TextPost', ['id', 'title', 'text', 'channel', 'api'])
-
-LinkPost = namedtuple('LinkPost', ['id', 'title', 'url', 'channel', 'api'])
-
-Comment = namedtuple('Comment', ['id', 'text', 'comment_id', 'children', 'post_id', 'api'])
+class Channel:
+    """Simple factory representation for a channel"""
+    def __init__(self, **kwargs):
+        self.name = kwargs.get('name', None)
+        self.title = kwargs.get('title', None)
+        self.channel_type = kwargs.get('channel_type', None)
+        self.public_description = kwargs.get('public_description', None)
+        self.api = kwargs.get('api', None)
+
+
+class Post:
+    """Simple factory representation for a post"""
+    def __init__(self, **kwargs):
+        self.id = kwargs.get('id', None)
+        self.title = kwargs.get('title', None)
+        self.text = kwargs.get('text', None)
+        self.url = kwargs.get('url', None)
+        self.channel = kwargs.get('channel', None)
+        self.api = kwargs.get('api', None)
```

Comment:
should it have `created` here as well?

Reply:
Or does it now always get filled in with a value from the API request to reddit? Just wondering why `id` is here but `created` isn't, since they both are values that we don't set (i.e. they come back from reddit).

Reply:
Good catch, it should for consistency so the property is always present.